Firestore/core/src/firebase/firestore/remote/grpc_stream_operation.h

```diff
@@ -48,16 +48,12 @@ class GrpcStream;
  * Operation expects all GRPC objects pertaining to the current stream to remain
  * valid until the operation comes back from the GRPC completion queue.
  */
-class StreamOperation : public GrpcOperation {
+class GrpcStreamCompletion : public GrpcOperation {
```

Comment:
I don't see much value in the difference between `GrpcStreamCompletion` and `GrpcOperation` now. Can we merge them and just call this `GrpcCompletion`?